src/main/java/hotelreservation/domain/Reservation.java

```diff
@@ -0,0 +1,73 @@
+package hotelreservation.domain;
+import javax.persistence.*;
```

Comment:
May be best to only include relevant bits from persistence lib

Reply:
it works so we can add it to the backlog.  For this program I don't think making that edit would really do that much for us.